docs/useful_tools.md

```diff
@@ -86,21 +86,29 @@ Or you can use 3D visualization software such as the [MeshLab](http://www.meshla
 
 ## Dataset
 
-To browse the KITTI directly without inference, you can run the following command
+We also provide scripts to visualize the dataset without inference. You can use `tools/misc/browse_dataset.py` to show loaded data and ground-truth online and save them on the disk. Currently we support all the single-modality 3D detection, KITTI and SUN RGB-D multi-modality 3D detection and all the 3D segmentation datasets. To browse the KITTI dataset, you can run the following command
```

Comment:
we support single-modality 3D detection and 3D segmentation on all the datasets, multi-modality 3D detection on KITTI and SUN RGB-D.

Reply:
Perfect!